spec/index.bs

```diff
@@ -12308,23 +12379,38 @@ interface GPUCanvasContext {
 </dl>
 
 <div algorithm>
-    In the "update the rendering or user interface of that `Document`" sub-step of the
-    "[=Update the rendering=]" step of the HTML processing model, each {{GPUCanvasContext}} |context|
-    must <dfn abstract-op>update the rendering of the WebGPU canvas</dfn> by running the following steps:
+    To <dfn abstract-op>commit the canvas texture</dfn> for {{GPUCanvasContext}} |context|:
 
     1. Update the rendering of |context|.{{GPUCanvasContext/canvas}} with
         [$get a copy of the image contents of a context|a copy of the image contents$] of |context|.
+
+        Note: The image still won't be presented until the whole document's rendering is updated
+        in "[=Update the rendering=]".
     1. If |context|.{{GPUCanvasContext/[[currentTexture]]}} is not `null`:
         1. Call |context|.{{GPUCanvasContext/[[currentTexture]]}}.{{GPUTexture/destroy()}}
             (without destroying |context|.{{GPUCanvasContext/[[drawingBuffer]]}})
             to terminate write access to the image.
         1. Set |context|.{{GPUCanvasContext/[[currentTexture]]}} to `null`.
 
+    Note:
+    This will be called multiple times for the same texture, in some order, from both
+    {{GPUCanvasContext/getCurrentTexture()}} and [$update the rendering of the WebGPU canvas$].
+    The second call has no effect.
```

Comment:
Actually, re: my comment above, I wonder if this is enough? Just acknowledging that we might enter this algorithm multiple times for a single texture and that's OK? I'd feel better about that than indicating that the expiration step can be skipped.